int8                        robot_id
geometry_msgs/PoseStamped   robot_pose
int8[]                      auction_ids
geometry_msgs/PoseStamped[] poses

